Xamarin.Forms.Platform.UAP/WindowsIsolatedStorage.cs

```diff
@@ -4,13 +4,15 @@
 using Windows.Storage;
 using Windows.Storage.FileProperties;
 using Windows.Storage.Streams;
-using Xamarin.Forms.Internals;
+//using FileMode = Xamarin.Forms.Internals.FileMode;
```

Comment:
Do we need these here?

Reply:
Yes, we will try to get Netstandard1_0 after on UAP and will need this